.github/workflows/release-page.yml

```diff
@@ -23,6 +23,8 @@ jobs:
   create_github_release:
     if: ${{ inputs.type }} == 'patch'
     environment: oidc_aws_s3_upload
+    needs:
+      - glvd
```

Comment:
For readability, I would suggest to place the definitions of the jobs in the order they run:
1. glrd
2. glvd 
3. create_github_release